lighthouse-core/gather/connections/connection.js

```diff
@@ -117,7 +121,7 @@ class Connection {
     if (callback) {
       this._callbacks.delete(object.id);
 
-      return callback.resolve(Promise.resolve().then(_ => {
+      callback.resolve(Promise.resolve().then(_ => {
```

Comment:
this seems like a mistake...

Reply:
There's nothing to return, the method returns void.

Reply:
oh wow that was quite the important diff elided I missed 😆 it jumped straight from 
```
   * @return {Promise<LH.CrdpCommands[C]['returnType']>}
   */	   */
  sendCommand(method, ...paramArgs) {	  sendCommand
```

to this, my b